src/test/java/se/sundsvall/eventlog/api/EventResourceTest.java

```diff
@@ -53,10 +53,10 @@ void getEvent() {
 
 	private Event createEvent() {
```

Comment:
As EventService now is in place, add verifications that it is called in each tests (i.e. replace the todos in this class and in the EventResourceFailureTest).